als-server/shared/src/main/scala/org/mulesoft/als/server/modules/workspace/WorkspaceContentManager.scala

```diff
@@ -340,9 +343,8 @@ class WorkspaceContentManager private (val folderUri: String,
     cacheConfig
       .parse(decodedUri)
       .map { r =>
-        // temporal registration until new registration is implemented, this should work as with AMF 4.x.x
         r.result.baseUnit match {
-          case d: Dialect =>
+          case d: Dialect if hotReloadDialects =>
```

Comment:
isn't hotReloadDialects always false?

Reply:
yes, as of today this is not configurable, but this is in order to be future-proof, there were talks about allowing this kind of behavior